index.js

```diff
@@ -37,10 +37,17 @@ export const startEvaluate = async ({
     if (!rounds.current) {
       rounds.current = new RoundData(roundIndex)
     } else if (rounds.current.index !== roundIndex) {
-      // This should never happen
-      throw new Error(
-        `Round index mismatch: ${rounds.current.index} !== ${roundIndex}`
-      )
+      // This occassionally happens because of a race condition between onMeasurementsAdded
+      // and onRoundStart event handlers. We should eventually fix it.
```

Comment:
```suggestion
      // This occassionally happens because of a race condition between onMeasurementsAdded
      // and onRoundStart event handlers. https://github.com/filecoin-station/spark-evaluate/issues/233
```